src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String HEAD_MESSAGE = "Report for period %s - %s" + System.lineSeparator();
+    private static final String BODY_MESSAGE = "%s - %s";
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder report = new StringBuilder();
+        int[] totalSalary = new int[names.length];
+        int nameIndex = 0;
```

Comment:
Make all the indexes constants
![image](https://github.com/mate-academy/jv-salary-info/assets/20326858/7baace0a-8933-4b52-86d0-7fa4fba3c8e3)

Reply:
nameIndex is dynamic, so it cant be constant.